sdk/maps/azure-maps-search/azure/maps/search/_shared/converter.py

```diff
@@ -0,0 +1,113 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+
+"""
+Maps SDK converter for Geo Json
+This module provides converters to and from GeoJSON geo_interface, etc.
+Azure Maps Search uses these converters; they can also be used directly.
+"""
+import json
+import re
+from ast import literal_eval
+from typing import Union
+
+
+geo_types = {
+    'Point',
+    'MultiPoint',
+    'LineString',
+    'MultiLineString',
+    'Polygon',
+    'MultiPolygon'
+}
+
+wkt_types = {x.upper() for x in geo_types}
```

Comment:
This looks great, but where are the unit tests and documentation? You mention being about to use `__geo_interface__` but I don't see anywhere you talk about using WKT. Also, what about WKB (well-known binary)?  Not crucial to add it now since it would be a non-breaking additive change.

Reply:
I think I will just mainly use `__geo_interface__` and GeoJson object for now. Maybe after public review we can see if users prefer to use WKB as well